config.py

```
@@ -0,0 +1,191 @@
+        Returns:
+            MCPConfig: Configuration instance with environment overrides
+        """
+        config = cls()
+        
+        # Load timeout settings
+        if 'ESP_MCP_TIMEOUT' in os.environ:
+            try:
+                config.DEFAULT_COMMAND_TIMEOUT = int(os.environ['ESP_MCP_TIMEOUT'])
+                logger.debug(f"Loaded timeout from env: {config.DEFAULT_COMMAND_TIMEOUT}s")
+            except ValueError:
+                logger.warning(f"Invalid ESP_MCP_TIMEOUT value, using default")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)